app/views/play/level/ControlBarView.coffee

```diff
@@ -79,6 +78,10 @@ module.exports = class ControlBarView extends CocoView
     @render()
 
   onLoaded: ->
+    if @inExam
+      pnum = userUtils.levelNumberInExam(@level.get('slug'))
+      @setLevelName($.i18n.t('exams.level_num', { num: pnum }))
```

Comment:
they came back and they want it to look like Challenge X instead of Level X? Can you make that change?

I will make the change on the exam page

Reply:
oh you don't need to change on exam page. this uses same i18n so I can change it